qa/tasks/mgr/dashboard/test_requests.py

```diff
@@ -21,3 +21,9 @@ def test_force_no_gzip(self):
         self.assertHeaders({
             'Content-Type': 'application/json',
         })
+
+    def test_server(self):
+        self._get('/api/summary')
+        self.assertHeaders({
+            'server': 'Ceph-Dashboard'
+        })
```

Comment:
I think it expects a new empty line at the end but its okay I think

Reply:
Might be, but the previous one didn't have it either... In any case, as it's been a clean backport 🥳 let's not modify it!

Reply:
Yup lets not.